infrastructure/src/main/java/org/corfudb/infrastructure/log/StreamLogFiles.java

```diff
@@ -120,15 +121,15 @@ public StreamLogFiles(ServerContext serverContext, boolean noVerify) {
         channelsToSync = new HashSet<>();
         this.verify = !noVerify;
         this.dataStore = StreamLogDataStore.builder().dataStore(serverContext.getDataStore()).build();
-        String logSizeQuotaParam = (String) serverContext.getServerConfig().get("--log-size-quota-bytes");
+        String logSizeQuotaParam = (String) serverContext.getServerConfig().get("--log-size-quota-percentage");
 
-        logSizeLimit = Long.parseLong(logSizeQuotaParam);
+        logSizeLimitPercentage = Double.parseDouble(logSizeQuotaParam);
```

Comment:
rename logSizeQuotaParam to another name? The name of 'logSizeQuota' has been used for resource

Reply:
done.